### ISSUE TITLE
The auto capture function of the tool..

### ISSUE THREAD

**Issue by c933103** [ShareX/ShareX#3080]
.... a few days ago, I was going to try the function of automatically make a capture of the screen after x seconds, which I thought I failed but actually it succeeded, which generated a few problems:
1. these auto-screenshots mixed together with my regular screenshots that I am not sure how to separate them
2. these auto-screenshots get automatically uploaded onto the cloud image hosting and I am not sure how to delete them
3. also I am not sure about how to delete them from ShareX's history
And that lead to a few more suggestions:
4. Would it be possible to have on screen visual indication that auto capture is ongoing at least have the option for such an indication to show up and especially when capture region is not full screen
5. Would it be possible to have an option for audio notification when each of these periodic capture task a done
6. Would it be possible to have an option in ShareX toolbar menu that tell users such capture is occuring and also provide a button to quickly stop such a capture?
7. Would it be possible to have a different set of post-capture action for these automatic tasks than regular uploads?

**Comment by Jaex** [comment]
When auto capture running there is new tray icon, which shows it is running and allows you to stop it by clicking icon:

![](https://jaex.getsharex.com/2018/01/huDzu3bnfi.png)

**Comment by vatterspun** [comment]
> these auto-screenshots get automatically uploaded onto the cloud image hosting and I am not sure how to delete them

I think you can right-click on the images, right-click, and select deletion URL.  You'll then have to visit each of these URLs, which admittedly might take some time.

![image](https://user-images.githubusercontent.com/1289134/35188980-e6d3bef2-fe06-11e7-8bac-825c583c243a.png)

> Would it be possible to have a different set of post-capture action for these automatic tasks than regular uploads?

I think it's possible to set that up via the hotkey creation view


> when recording a screen, there is a timer showing how much seconds have been recorded. Would it be possible to add a similar timer for autocapture showing how many seconds left for next capture?

Do you expect, if you do full screen auto-capture, that this timer get captured?

**Comment by c933103** [comment]
>Use task setting to override after-capture setting and a stop button

I think it would be a good idea if these settings are integrated to the menu for the default auto capture task

>Do you expect, if you do full screen auto-capture, that this timer get captured?

It can optionally be shown and disappear when the timer reach zero

**Comment by vatterspun** [comment]
> I think it would be a good idea if these settings are integrated to the menu for the default auto capture task

Can you suggest how you picture that getting implemented?

**Comment by wolfen351** [comment]
I'd like to add my 2c here - this is a wishlist, feature request:

* It would be really nice to have a different subfolder within the default location for these images, or at least a different file name structure: AutoCapture_2001-01-01.jpg or similar - this allows me to later do things like make a time lapse video of my screen without the regular screenshots interfering. - I can imagine a extra textbox in the auto capture url with a browse button to set the root storage path. 

* It would be great to have a similar menu to the right click "after capture" menu that allows you to select tasks, i can imagine "after timed screenshot" with the same stuff inside. 

Basically treat the timed screenshots as another class of screenshot that has different actions and tasks associated.

**Comment by wolfen351** [comment]
Here is the menu I wish to see duplicated for the timed screenshots:

![image](https://user-images.githubusercontent.com/5477547/36867586-757d3c38-1dfa-11e8-8e8f-2ec24cf76cc8.png)

Here is the dialog I wish to see an option for setting a root path: 

![image](https://user-images.githubusercontent.com/5477547/36867649-9dd810a4-1dfa-11e8-9685-5383b64cd936.png)

**Comment by oxysoft** [comment]
This should be reopened.
